authorization/1.6/_index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Jakarta Authorization 1.6"
+date: 2019-08-07
+summary: "First release for Jakarta EE 8"
+---
+Jakarta Authorization defines a low-level SPI for authorization modules, which are repositories of permissions 
+facilitating subject based security by determining whether a given subject has a given permission, and algorithms
+to transform security constraints for specific containers (such as Jakarta Servlet or Jakarta Enterprise Beans) into 
+these permissions.
+
+* [Jakarta Authorization 1.6 Specification Document](./authorization-spec-1.6.pdf) (PDF)
+* [Jakarta Authorization 1.6 Specification Document](./authorization-spec-1.6.html) (HTML)
+* [Jakarta Authorization 1.6 Javadoc](./apidocs)
+* [Jakarta Authorization 1.6 TCK](https://download.eclipse.org/ee4j/jakartaee-tck/jakartaee8-eftl/promoted/eclipse-authorization-tck-1.5.0.zip)
```

Comment:
This needs to be the URL of the final signed version, which will be at:
https://download.eclipse.org/jakartaee/authorization/1.5/eclipse-authorization-tck-1.5.0.zip